xl/xenops_server_xenlight.cppo.ml

```diff
@@ -2208,7 +2208,7 @@ module VM = struct
 								boot = Some hvm_info.Xenops_interface.Vm.boot_order;
 								usb = Some true;
 								usbdevice_list = [ "tablet" ];
-#if xen45
+#if xen45 = 1
```

Comment:
Here you would use `#ifdef xen45` if you were following the alternative approach.